client/src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import RecommendedAlbums from './components/RecommendedAlbums.jsx';
+
+
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      albumResults: [],
+      albumTags: null,
+      artist: null
+    }
+    this.getRelatedAlbums = this.getRelatedAlbums.bind(this);
+    this.getExampleAlbumInfo = this.getExampleAlbumInfo.bind(this);
+  }
+
+  componentDidMount() {
+    this.getRelatedAlbums()
+    this.getExampleAlbumInfo()
+  }
+
+  getRelatedAlbums() {
+    fetch(`/api/albums/${this.props.album.id}`)
+      .then(response => {
+        return response.json();
+      })
+      .then(albumResults => {
+        this.setState({ albumResults });
+      });
+  }
+
+  getExampleAlbumInfo() {
+    fetch(`/api/album/${this.props.album.id}`)
+    .then(response => {
+      return response.json();
+    })
+    .then(album => {
+      this.setState({artist: album.artist, albumTags: album.tags})
+    })
+  }
+
+  render() {
+    console.log(this.state.albumResults)
```

Comment:
If you have the React dev tools installed, it shows you the state really easily. Might be a better bet for debugging purposes in the future. 👍